src/involvement/models/role.py

```diff
@@ -153,7 +153,7 @@ def editable_role_types(user):
         elif is_fum(user):
             return ['board']
         elif is_board(user):
-            return ['presidium']
+            return ['presidium', 'group_leader', 'engaged']
```

Comment:
These two functions that you have changed here are identical. It would be nice if you could merge the together into one function so that we avoid repetition

Reply:
We discussed this and decided on leaving it as is.